### PR TITLE
Json msg

### PR DESCRIPTION
**Pull request by bibiebliss**
i think i made the same mistake again because this is showing the commit of another branch no? idk
Screen Shot 2021-06-12 at 2 15 22 PM

### CODE REVIEW

**Comment by bibiebliss** [comment]
Screen Shot 2021-06-12 at 2 26 56 PM

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -5,14 +5,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    String[] strings = {"Pursued a research project with a professor on computational models of human category learning where we explored existing models based on deep neural networks",
```
Comment (logannc) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:13]: don't use array primitives, prefer ArrayList

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 2]
```diff
@@ -5,14 +5,26 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello... this will turn into something much more fascinating soon!");
+    String json = convertToJsonUsingGson(strings);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+  private String convertToJsonUsingGson(String[] list) {
```
Comment (logannc) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:25]: this can be static

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -81,9 +91,9 @@ <h2 class="title">About Me</h2>
         <div class="max-width">
             <div class="facts-content">
                 <div class="left">
-                    <p>Click here to get a random fun fact about The Office:</p>
-                    <button onclick="addRandomGreeting()">The Office Fact</button>
-                    <div id="greeting-container"></div>
+                    <p>Click here to get a random work/research experience of mine!</p>
+                    <button onclick="showContent()">Click Me!</button>
```
Comment (logannc) [inline portfolio/src/main/webapp/index.html:95]: nit: just a style difference, but I prefer adding event listeners over stringy-javascript-in-html. e.g.,.

```
let theButton = document.querySelectorAll('button#showContentButton')[0];
theButton.addEventListener('click', showContent);
```

Obviously, you'd need to add the id and double check I didn't do something silly here.

**Review by logannc — APPROVED** [review]

**Pull request merged**